Euler_discret.q

```diff
@@ -0,0 +1,49 @@
+\l qlib.q
+\l transform.q

```

Comment:
In case you move transform to qlib you can use 
q) .import.module `transform
to load it.

How long have you spent time on this exercise?